web/test/pages/profile/profile-foreign.test.tsx

```diff
@@ -543,4 +543,31 @@ describe("profile-foreign", () => {
       expect(screen.queryByTestId("elevatorOwningBusiness-radio-group")).not.toBeInTheDocument();
     });
   });
+
+  it("does not render profile address fields when business persona is FOREIGN", () => {
```

Comment:
pebble - what's the value of this test? If we follow red-green-refactor approach, this test is passing from the get go